### PR TITLE
[NUI] Remove the legacy code of AccessibilityManager

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
### Description of Change ###
- `AccessibilityManager` hasn't been used anymore after Tizen 7.0 version.
- Now, all the legacy codes of AccessibilityManager should be removed in order to prevent the confusion.
- All Accessibility APIs are not public ones because they have never done ACR before, and that's why we can remove the codes like this.

* dali related patch : https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/319388/

### API Changes ###
- APIs related to `AccessibilityManager` class
- Some APIs are removed, but they're not public ones on the platform.

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs [hunk 1]
```diff
@@ -1810,17 +1807,6 @@ internal void BaseRemove(View view)
             base.Remove(view);
         }
 
-        internal override bool OnAccessibilityPan(PanGesture gestures)
```
Comment (youngsuns) [inline src/Tizen.NUI.Components/Controls/ScrollableBase.cs:1813]: QQ. PanGesture 관련해서 현재 쓰이는 코드는 어떤건가요?

**Review by youngsuns — APPROVED** [review]
Looks good!

**Review by hinohie — APPROVED** [review]
Looks good to me!

**Comment by hinohie** [comment]
Additional info : Their is no code who use AccessibilityManager in VD now.

Search at https://code.sec.samsung.net/codegrok/ TIZEN_APP/ and TIZEN/[MAIN]/[TIZENPROD_Prj]/[INT]/

File: src/Tizen.NUI.Components/Controls/ScrollableBase.cs (hunk 1 above)
Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Components/Controls/ScrollableBase.cs:1813]: 답이 늦어 죄송합니다 

`OnAccessibilityPan`는 기존 AccessibilityManager가 사용될 시기에 Window system 쪽에서 불리던 함수였습니다.

아래 EventHandler에서 gesture를 받아
https://github.sec.samsung.net/tizen/dali-adaptor/blob/tizen_6.0_snapshot/dali/internal/window-system/common/event-handler.cpp#L382

Scroll gesture (TwoFingersHover)를 받으면, 아래 Detector에게 event를 send하고 Detector는 event condition에 따라 Panning event를 보내줍니다.
https://github.sec.samsung.net/tizen/dali-adaptor/blob/tizen_6.0_snapshot/dali/internal/accessibility/common/accessibility-gesture-detector.cpp#L95

이를 override한 해당 함수는, 특정 조건(page 넘기는 앱 or scroll이 진행 중일때 등)이 아닐 때는 항상 일반 panning 처럼 동작하도록 `OnPanGesture()`를 부르도록 했습니다.

**Comment by Seoyeon2Kim** [comment]
> Additional info : Their is no code who use AccessibilityManager in VD now.
> 
> Search at https://code.sec.samsung.net/codegrok/ TIZEN_APP/ and TIZEN/[MAIN]/[TIZENPROD_Prj]/[INT]/

Thank you for checking the latest code of VD team! :)
There are a few remaining codes in DA still.. So, I'll let them know about the change.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**